projects/netgrif-components-core/src/lib/data-fields/file-list-field/file-list-default-field/abstract-file-list-default-field.component.ts

```diff
@@ -167,6 +168,17 @@ export abstract class AbstractFileListDefaultFieldComponent extends AbstractBase
             }
         }
 
+        if (this.dataField.allowTypes && this.dataField.allowTypes?.length > 0) {
+            for (let i = 0; i < this.fileUploadEl.nativeElement.files.length; i++) {
```

Comment:
``this.fileUploadEl.nativeElement.files`` is repeated 4x in this algorithm, please consider to extract it to a const for better code readability.

Reply:
done